### PR TITLE
Compare content

### PR DESCRIPTION
**Pull request by Actionmaker1**
Content filled in the Compare view

### CODE REVIEW

File: app/assets/stylesheets/pages/_home.scss [hunk 1]
```diff
@@ -1,29 +1 @@
```
Comment (CharlineNoga) [inline app/assets/stylesheets/pages/_home.scss]: I believe this can be a conflict with Dan´s last push...

File: app/views/pages/home.html.erb [hunk 2]
```diff
@@ -63,19 +63,17 @@
     <a href="welcome" type="button" class="btn btn-primary">Shall we start?</a>
 
 </div>
-<footer class="footer_wide">
```
Comment (CharlineNoga) [inline app/views/pages/home.html.erb]: same here

File: app/assets/stylesheets/components/_footer.scss [hunk 3]
```diff
@@ -1,7 +1,5 @@
```
Comment (CharlineNoga) [inline app/assets/stylesheets/components/_footer.scss:1]: and here...

**Review by CharlineNoga — APPROVED** [review]

**Comment by dwiehoff** [comment]
Has been merged in another PR

**Pull request closed without merging**